src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(human => (
+    century
+      ? human.sex === 'm'
+        && Math.ceil(human.died / 100) === century
+      : human.sex === 'm'
+  ));
+
+  const menAges = men.map(man => (man.died - man.born));
+  const menAgesSum = menAges.reduce((prev, curr) => prev + curr);
```

Comment:
1 - you can just use reduce, no need to use map too (just calculate age in reduce method)
2 - you've used same logic in next function, so better to move it to separate function and reuse it

Reply:
fix it everywhere